<launch>
    
    <!-- for global mapping -->
    <arg name="out_points_topic" default="/occupied_point" />
    <arg name="out_pose_topic" default="/map_pose" />
    <arg name="out_file" default="/home/cc/chg_ws/ros_ws/semantic_map_coda_ws/src/dsp_global_mapping/data/result.ply" />
    <!-- <arg name="map_range_xy" default="61.44" />
    <arg name="map_range_z" default="30.72" />
    <arg name="voxel_size" default="0.12" /> -->
    <arg name="map_range_xy" default="38.4" />
    <arg name="map_range_z" default="38.4" />
    <arg name="voxel_size" default="0.15" />
    <arg name="write_color" default="0" />
    <arg name="object_csv" default="/home/cc/chg_ws/ros_ws/semantic_map_coda_ws/src/dsp_global_mapping/cfg/object_info_kitti360.csv" />

    <node pkg="dsp_global_mapping" type="global_mapping" name="global_mapping" output="screen" args="$(arg out_points_topic) $(arg out_pose_topic) $(arg out_file) $(arg map_range_xy) $(arg map_range_z) $(arg voxel_size) $(arg write_color) $(arg object_csv)" />

</launch>